Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -45,12 +48,12 @@ function verify_vmbus_interrupts() {
     #
     # Verify if VMBUS interrupts are processed by all CPUs
     #
-    while read line
+    while read -r line
     do
         if [[ ($line = *hyperv* ) || ( $line = *Hypervisor* ) ]]; then
-            for (( core=0; core<=$cpu_count-1; core++ ))
+            for (( core=0; core<=${cpu_count-1}; core++ ))
             do
-                intrCount=$(echo "$line" | xargs echo |cut -f $(( $core+2 )) -d ' ')
+                intrCount=$(echo "$line" | xargs echo | cut -f $(( ${core+2} )) -d ' ')
```

Comment:
I don't quite understand what you're trying to accomplish here. Do you want to print the `$core + 2` column of `line`?

Reply:
yes

Reply:
And is it working? I didn't know that you could do arithmetic inside `${}`, only inside `$(())`.  I tried it here and this doesn't seem to be what you're looking for:

```
bash-3.2$ var=123
bash-3.2$ echo ${var+2}
2
bash-3.2$ echo ${var+300}
300
```

The original code, however, seems to be correct here:

```
bash-3.2$ echo $((var+2))
125
bash-3.2$ echo $((var+300))
423
```

Reply:
Thanks, I will fix it.